test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SWAEndToEndTests.cs

```diff
@@ -67,7 +70,7 @@ public async Task InvokeFunction_FunctionLevel_ValidToken_Succeeds(string header
         {
             // if an admin token is passed, the function invocation succeeds
             HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "api/HttpTrigger-FunctionAuth?code=test");
-            string token = _fixture.Host.GenerateAdminJwtToken();
+            string token = GetSWAAdminJwtToken();
```

Comment:
After updating the test fixture such that AzureWebEncryptionKey and WEBSITE_AUTH_ENCRYPTION_KEY were different, this existing test failed when sending a token signed with AzureWebEncryptionKey.